scipy/signal/tests/test_cont2discrete.py

```diff
@@ -33,6 +33,28 @@ def test_zoh(self):
         assert_array_almost_equal(dc, dd)
         assert_almost_equal(dt_requested, dt)
 
+    def test_foh(self):
```

Comment:
perhaps a brief comment describing the source of the "expected" values could be helpful?